LayoutKit/Internal/CGFloatExtension.swift

```diff
@@ -0,0 +1,29 @@
+// Copyright 2016 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import UIKit
+
+extension CGFloat {
+
+    /**
+     Returns the current float rounded up to the nearest fraction of a pixel
+     that the screen density supports.
+     */
+    var roundedUpToFractionalPoint: CGFloat {
+        if self <= 0 {
+            return 0
+        }
```

Comment:
this documentation doesn't say that the result would be forced to be non-negative